WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/utils/DateUtils.kt

```diff
@@ -28,7 +30,7 @@ class DateUtils @Inject constructor(
         }
 
         val formatter = SimpleDateFormat("EEEE", localeManagerWrapper.getLocale())
-        return formatter.format(c.time).capitalize()
+        return formatter.format(c.time).capitalize(Locale.ROOT)
```

Comment:
AFAIU this string will be presented to the user, so it shouldn't use the ROOT locale. Moreover, there is `localeManagerWrapper.getLocale()` another getter for locale two lines above. Wdyt?

Reply:
Oh, this was accidentally left while I was trying to figure out the issue with capitalize. I initially used `localeManagerWrapper.getLocale()` but since I got the lint error, I was just trying to figure out why it still was complaining. Sorry about that, I'll fix it once we figure out what the lint issue is.

Reply:
Fixed in 7bcf135e001a8998be13773cdd5cb444bf98fbdd. I opted to go with `localeManagerWrapper.getLocale()` because I think `LocalManagerWrapper` should be the one calling the `getSafeLocale` if necessary. Otherwise we'd be bypassing the provided `LocaleManagerWrapper` which would be unexpected by the caller of this method.